scalar.c

```diff
@@ -886,7 +911,7 @@ static int cmd_clone(int argc, const char **argv)
 			cache_server_url = default_cache_server_url;
 		if (set_config("core.useGVFSHelper=true") ||
 		    set_config("core.gvfs=150") ||
-		    set_config("http.version=HTTP/1.1")) {
+		    set_config("http.%s.version=HTTP/1.1", url)) {
```

Comment:
This should be sufficient for answering my concerns for `scalar clone` commands against ADO. Thanks.

Reply:
@dscho: Somehow the comment I tried to leave yesterday didn't come through. I must have started a review that didn't get complete.

This line won't actually be sufficient because it's replacing a wider-focus config change for `http.version=HTTP/1.1`. The issue is that _before_ this change there is a call to `supports_gvfs_protocol()` that will fail if this config isn't set.

Sorry, this PR doesn't actually resolve the issue due to this problem. Hard to recognize when the diff isn't there and the tests do not cover a real-world example.